### PR TITLE
M4

### PR DESCRIPTION
**Pull request by rgozim**

### CODE REVIEW

**Comment by rgozim** [comment]
Hmm some irregularities in this PR. I'll need to reviewer further. Please do not merge.

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ From a high level, blitz-plugin consists of the following tasks/stages:
 
 1. Import `.ome.xml` map files from `org.openmicroscopy:omero-model` (`omero-model.jar`) resources
 2. Import `-types.properties` files from `org.openmicroscopy:omero-model` (`omero-model.jar`) resources
-3. Using the [`omero-dsl-plugin`](https://gitlab.com/openmicroscopy/incubator/omero-dsl), generate `xx.combined` files
+3. Using the [`omero-ome.dsl-plugin`](https://gitlab.com/openmicroscopy/incubator/omero-ome.dsl), generate `xx.combined` files
```
Comment (jburel) [inline README.md]: This should now point to the gh repo

**Comment by joshmoore** [comment]
And another one green! @rgozim, note that this one required a commit that I wasn't expecting: https://github.com/ome/omero-blitz-plugin/pull/6/commits/69c46b127c330d907e8072f5d2ddece22576ac7b

Leaving to you to decide whether or not to roll that back in the next go-round. Merging and releasing.

**Comment by joshmoore** [comment]
Oops, strike that:

```
/opt/omero-gradle-plugins/omero-blitz-plugin $git grep SNAPSHOT
src/main/groovy/org/openmicroscopy/blitz/ImportHelper.groovy:                dependencies.add(project.getDependencies().create("org.openmicroscopy:omero-model:5.5.0-SNAPSHOT"))
```

Think this needs to get discussed before I progress.

**Pull request merged**